### PR TITLE
#8556 add two different disease test results for one sample

### PR DESCRIPTION
**Pull request by richardbartha**
Create automated test to check adding 2 positive Pathogen Test Result of different diseases to a Sample for cases, contacts and event participants

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java [hunk 1]
```diff
@@ -843,16 +847,6 @@ public EventDirectorySteps(
               getCheckboxByUUID(apiState.getCreatedEvent().getUuid()));
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
-
-    When(
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java:847]: why removed? might be used in other tests

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java [hunk 2]
```diff
@@ -119,6 +195,71 @@ public CreateNewSampleSteps(
...
+        "^I validate only one sample is created with two pathogen tests",
+        () -> {
+          softly.assertEquals(
+              webDriverHelpers.getNumberOfElements(EDIT_SAMPLE_BUTTON),
+              1,
+              "Number of samples is not correct");
+          softly.assertEquals(
+              webDriverHelpers.getTextFromWebElement(NUMBER_OF_TESTS_IN_SAMPLES),
+              "Number of tests: 2",
+              "Number of tests is correct!");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java:251]: assert all needed

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java [hunk 3]
```diff
@@ -119,6 +195,71 @@ public CreateNewSampleSteps(
...
+        });
+
+    When(
+        "^I validate the existence of two pathogen tests",
+        () -> {
+          softly.assertEquals(
+              webDriverHelpers.getNumberOfElements(EDIT_PATHOGEN_TEST),
+              2,
+              "Number of pathogen tests is not correct");
+        });
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java:261]: assert all needed

**Review by rdutu-vg — APPROVED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java (hunk 1 above)
Comment (richardbartha) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java:862]: was removing duplicates and this was deleted by mistake i added it back

**Pull request merged**